java/semantickernel-core/src/main/java/com/microsoft/semantickernel/templateengine/blocks/ValBlock.java

```diff
@@ -4,88 +4,63 @@
 
 import com.microsoft.semantickernel.orchestration.ContextVariables;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import reactor.util.annotation.Nullable;
 
 public class ValBlock extends Block implements TextRendering {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ValBlock.class);
+
+    // Cache the first and last char
+    private char first = '\0';
+    private char last = '\0';
+
+    // Content, excluding start/end quote chars
+    private String value = "";
+
     public ValBlock(String content) {
```

Comment:
`content` is optional

In c# this variable is named `quotedValue`, any reason to change this?